android/src/main/java/com/tanguyantoine/react/MusicControlNotification.java

```diff
@@ -50,21 +52,22 @@ public void show(NotificationCompat.Builder builder, boolean isPlaying) {
         if(stop != null) builder.addAction(stop);
         if(next != null) builder.addAction(next);
 
-        // Notifications of playing music can't be removed
-        builder.setOngoing(isPlaying);
+        // Set whether notification can be closed based on closeNotification control (default PAUSED)
+        if (this.module.notificationClose == MusicControlModule.NotificationClose.ALWAYS) builder.setOngoing(false);
```

Comment:
Use the same code style here by adding the brackets

Also, remove `this` to make it cleaner.